shared/constants/chat2/convostate.tsx

```diff
@@ -1520,13 +1520,14 @@ const createSlice: Z.ImmerStateCreator<ConvoState> = (set, get) => {
             s.messageMap.delete(m.ordinal)
             s.messageTypeMap.delete(m.ordinal)
           } else {
-            // we sent it so lets keep our temp ordinal
-            if (m.outboxID && T.Chat.messageIDToNumber(m.id) !== T.Chat.ordinalToNumber(m.ordinal)) {
-              s.pendingOutboxToOrdinal.set(m.outboxID, m.ordinal)
```

Comment:
if we write here we could be overwriting something and that'll get stuck with no resolution